AmericanMedicalAssociation/AMA-style-guide
# EWL - 3112 animate audience selector

Ticket: https://issues.ama-assn.org/browse/EWL-3112

To test:
- Go to Organisms > Sections > Audience Selector
- If the viewport is less than 900px wide, the Audience Selector should collapse. You should be able to click on the dropdown and the list will slide animate to open. Clicking the list again will slide close.
- If the viewport is more than 900px wide, the Audience Selector should be visible and display inline.
